packages/neos-ui-editors/src/EditorEnvelope/index.js

```diff
@@ -78,7 +78,10 @@ export default class EditorEnvelope extends PureComponent {
             });
 
             return (
-                <EditorComponent className={classNames} id={this.generateIdentifier()} {...restProps} />
+                <EditorComponent className={classNames}
+                    id={this.generateIdentifier()}
+                    renderHelpIcon={this.renderHelpIcon.bind(this)}
```

Comment:
`.bind(this)` binds the context of the method to the current class - that means accesses to `this` inside of `renderHelpIcon` will resolve to `EditorEnvelope` instead of the context they are used in.

Reply:
An alternative to `.bind(this)` would be `() => this.renderHelpIcon()` - which preserves the current context (`EditorEnvelope`) since it is an array function.

Reply:
Heh, of course I know what bind does, I meant what's the justification of using it in this particular case, just missed that `renderHelpIcon` wasn't defined as an arrow function (which we normally do).

Maybe it makes sense to define `renderHelpIcon` as an arrow function itself here? https://github.com/neos/neos-ui/blob/2239fe5465a84971d6aa6a94ea2f7a058959bf59/packages/neos-ui-editors/src/EditorEnvelope/index.js#L140
We try not to use `.bind` and prefer arrow functions, for consistency's sake.

Reply:
Makes sense - I hadn't thought to define the render method as a arrow function directly. That is way cleaner. Will update the PR.